rpm/kubelet.spec

```diff
@@ -197,7 +197,10 @@ mv cni-plugins/bin/ %{buildroot}/opt/cni/
 
 
 %changelog
-* Tue Mar 20 2019 Lubomir I. Ivanov <lubomirivanov@vmware.com>
+* Tue Mar 28 2019 Lubomir I. Ivanov <lubomirivanov@vmware.com> - 1.14.1
+- Fix dependencies to CNI >= v0.6.0.
```

Comment:
If we merge this, then this line should also say ", as the project has decided it is up to the end user if they want to remove exposure to a CVE or not."